docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,355 @@
+
+Now, please run the example as the following script:
+```bash
+python train.py --cluster_mode yarn-cluster --remote_dir hdfs://path/to/remote/data
+```
+
+## 3.2 Run Programs with BigDL Provided Scripts
+For `spark-submit` users, BigDL provides a `bigdl-submit` script, you can find it in `${BIGDL_HOME}/scripts/bigdl-submit` and adjust the configurations according to your cluster settings. 
+
+```bash
+# bigdl-submit script
+spark-submit \
+  --properties-file ${BIGDL_CONF} \
+  --jars ${BIGDL_JARS} \
+  --conf spark.driver.extraClassPath=${BIGDL_JARS} \
+  --conf spark.executor.extraClassPath=${BIGDL_JARS} \
+  $*
+```
+
+In the `bigdl-submit` script:
+* It could automatically detect and setup configuration and jars files from the current conda environment, which will be set as `${BIGDL_CONF}` and `${BIGDL_JARS}` seperately. There is no need for you to setup these varaibles manully.
+* The BigDL configuration file will be uploaded to the cluster through `--properties-file`.
+* It will register BigDL dependency jars files to the Spark job through `--jars`.
+* `spark.driver.extraClassPath` will register the BigDL jars files to the driver classpath.
+* `spark.executor.extraClassPath` will register the BigDL jars files to the classpath of executors.
+
+Before submitting the Fasion-MNIST program to Yarn with `bigdl-submit`, we need to pack the Conda environment to an archive, which captures the Conda environment for Python and stores both Python interpreter and all its relevant dependencies.
+```bash
+conda pack -o environment.tar.gz#environment 
+```
+### Note:
+* If `environment.tar.gz` is not under the same directory with `train.py`, we should modify its path in `--archives` in the running command.
+
+### 3.2.1 Yarn Client
+The first thing we need to make sure is that the `cluster_mode` in `init_orca_context` must be `spark-submit` when using `bigdl-submit` to run Orca programs on yarn-client mode.
+
+```python
+from bigdl.orca import init_orca_context
+
+init_orca_context(cluster_mode="spark-submit")
+```
+
+When running the provided Fashion-MNIST program on yarn-client mode with `bigdl-submit`, we need:
+* Set the Python environment as the local Python path. For `yarn-client` mode, the Spark driver is running on local and it will use the Python interpreter in the current active conda environment.
+```bash
+--conf spark.pyspark.driver.python=/path/to/python #same as 'which python'
+```
+* Set the executor Python environment to the Python path in the Conda archive, since executors will use the Python interpreter in the conda archive.
+```bash
+--conf spark.pyspark.python=environment/bin/python
+```
+* Set the `archives` argument to the path of the Conda archive, which will be uploaded to remote clusters(i.e. HDFS) and distributed between executors.
+```bash
+--archives environment.tar.gz#environment
+```
+* Set the `py-files` argument to load and distribute dependency library on cluster.
+```
+--py-files orca_example.zip
+```
```

Comment:
we should not provide the zip file; instead, describe how to generate the zip file